### DISCUSSION TITLE
BirdNET GUI for Mac OS

### DISCUSSION THREAD

**Discussion by aaruljain** [birdnet-team/BirdNET-Analyzer#338] [Q&A]
I'm unable to launch the GUI for BirdNET-Analyzer for my Mac M2. Please help.

**Comment by Mattk70** [comment]
Why not try [Chirpity](https://chirpity.mattkirkland.co.uk)? That runs BirdNET on an M2 Mac.

**Reply by aaruljain** [reply]
Will that work if I want to train BirdNET classifiers also?

**Reply by Mattk70** [reply]
No. To do that, I believe you will need to run the BirdNET-Analyzer by following [these instructions](https://github.com/kahst/BirdNET-Analyzer?tab=readme-ov-file#setup-macos).
